### ISSUE TITLE
duplicate declaration of canSeeCookie()

### ISSUE THREAD

**Issue by Tscheffrey** [Codeception/Codeception#5500]
#### What are you trying to achieve?

I am trying to execute codeception with two comma-separated parameters:

```sh
./codecept.phar run -c ../.. --env fast,prod
```

#### What do you get instead?

The method `canSeeCookie()` appears to be generated twice in `_support/_generated/AcceptanceTesterActions.php`.

```sh
PHP Fatal error:  Cannot redeclare Company\Test\Site\_generated\AcceptanceTesterActions::canSeeCookie() in /Users/username/Sites/example.localhost/tests/_support/_generated/AcceptanceTesterActions.php on line 180
```
Everything works just fine if if execute it with only one --env parameter

### Details

* Codeception version: 3.0.0
* PHP Version: 7.0.33
* Operating System: macOS
* Installation type: Phar
* Suite configuration: 

`acceptance.suite.yml`:

```yml
class_name: AcceptanceTester
modules:
    enabled:
        - WebDriver
        - Asserts
```

`codeception.yml`:

```yml
namespace: Company\Test\Site
actor: Tester
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
settings:
    shuffle: true
extensions:
    enabled:
        - Codeception\Extension\RunFailed
modules:
    config:
        WebDriver:
            url: 'http://example.localhost'
            browser: chrome
env:
    local:
    stage:
        modules:
            config:
                WebDriver:
                    url: 'http://prev.example.de'
                    browser: chrome
    prod:
        modules:
            config:
                WebDriver:
                    url: 'http://example.com'
                    browser: chrome
    fast:
```

**Comment by diverent2** [comment]
Have you tried to rebuild you generated fuctions, using
```
$ codeception build
```
?
Also are you using any global declaration files like `acceptanceTester.php` or helpers?
Make sure your project doesn't include a native codeception function name by accident ex.
Having named a function the same as an `$I` function or another codeception function name.

**Comment by Tscheffrey** [comment]
> Have you tried to rebuild you generated fuctions, using
> 
> $ codeception build
> ?

yes, `AcceptanceTesterActions.php` seems to be generated properly, but on execution the file is being regenerated again with two versions of the method

**Comment by DanielRuf** [comment]
> with two versions of the method

Two exact same methods. It is actually duplicated.
I guess there are probably more duplicated methods then?

**Comment by diverent2** [comment]
Ok another update
generating the functions using 
```
$ ./tools/codecept.phar build -c ..
```
and then executing our files **with the option `--no-rebuild` enabled**
```
$ ./tools/codecept.phar run -c .. --env local,fast acceptance --no-rebuild
```
sets up the correct environment and works as expected.

I´m still pretty sure this is a bug.
Rebuilding the actor classes is also supposed to work using `-c` and multiple `--env`s

**Comment by Tscheffrey** [comment]
thanks @diverent2 , `--no-rebuild` works fine for me!

**Comment by Naktibalda** [comment]
Fixed in 3.0.1

**Comment by DanielRuf** [comment]
> Fixed in 3.0.1

Awesome =) Many thanks for taking the time to fix this and especially for maintaining Codeception 🙇
